### DISCUSSION TITLE
FCU cpflight

### DISCUSSION THREAD

**Discussion by CptBayer** [flybywiresim/aircraft#4215] [General]
Hi developers,

I have the FCU from cpflight. Any future plans to get this nice tool working with the flybywire mod?

![grafik](https://user-images.githubusercontent.com/76531893/113205028-187f8400-926e-11eb-88c7-c1b0770e7a90.png)

Greets Klaus

**Comment by tracernz** [comment]
Do you know how it interfaces with the sim? Does it work with the default asobo A320?

Looks very nice!

**Reply by CptBayer** [reply]
Hi,

no, not at all with Asobos 320. Guess I will contact Paolo from CPFlight and ask him ... This tool is a lot of fun!!! 

Greets Klaus

**Reply by CptBayer** [reply]
Hi,

CPflight uses FSUIPC to interface with a flightsimulator. No matter which one. FS 2020 has got not a lot of data which will read out by FSUIP. CPflight said, we have to have patinece and wait for more data .... Then they will possibly work on it to make it compatible with the FS 2020 ...

Greets Klaus

**Reply by CptBayer** [reply]
Hi,

I was in contact with Paolo from cpflight. His drivers are out now. We had a week of troubleshooting cause not all was working as it should. Paolos support was fantastic!!! Right now, still some very few issues left, but in general working very nice ... I love to fly with this FCU :-) ..

Greets Klaus
